cloud.ivanbotty.Launcher.yaml

```diff
@@ -0,0 +1,43 @@
+id: cloud.ivanbotty.Launcher
+runtime: org.gnome.Platform
+runtime-version: '49'
+sdk: org.gnome.Sdk
+command: cloud-ivanbotty-launcher
+
+# Permissions and filesystem access for the Flatpak app
+finish-args:
+  - --share=network                                 # Allow network access
+  - --filesystem=host                               # Access to host's directory (for indexing files)
+  - --socket=wayland                                # Enable Wayland display support
+  - --socket=fallback-x11                           # Enable fallback X11 support
+  - --share=ipc                                     # Required for X11 support
+  - --device=dri                                    # Allow direct rendering (GPU access)
+  - --talk-name=org.freedesktop.portal.Desktop       # Permission for Flatpak portals
```

Comment:
how many time do we have to tell NO. ?

https://github.com/flathub/flathub/pull/7408#discussion_r2649105749

Shall we just stop reviewing if you continue to ignore reviews.